modules/nw-networkpolicy-about.adoc

```diff
@@ -64,7 +64,8 @@ spec:
 If the Ingress Controller is configured with `endpointPublishingStrategy: HostNetwork`, then the Ingress Controller pod runs on the host network.
 When running on the host network, the traffic from the Ingress Controller is assigned the `netid:0` Virtual Network ID (VNID).
```

Comment:
This VNID assignment for traffic from the ingress controller only occurs in Openshift SDN I believe

Reply:
@astoycos , the upshot is that the whole page (when in HTML) is for OpenShift SDN, but google brings customers to this page even if the customer has OVN-K.  My intent was to clarify that this trick of labelling the `default` namespace applies to SDN only and there's no equivalent for OVN-K.

Reply:
Ahh ok Perfect, didn't realize that, thanks!